src/buffer/out/textBuffer.cpp

```diff
@@ -2265,6 +2267,48 @@ HRESULT TextBuffer::Reflow(TextBuffer& oldBuffer,
             CATCH_RETURN();
         }
 
+        // GH#32: Copy the attributes from the rest of the row into this new buffer.
+        // From where we are in the old buffer, to the end of the row, copy the
+        // remaining attributes.
+        // - if the old buffer is smaller than the new buffer, then just copy
+        //   what we have. as it was. The last attr in the row will be extended
+        //   to the end of the row in the new buffer.
+        // - if the old buffer is BIGGER, than we might have wrapped onto a new

```

Comment:
specifically "WIDER" not bigger. Xold > Xnew. It could be "TALLER" and it doesn't apply.